map_gen/maps/crash_site/commands.lua

```diff
@@ -29,10 +29,12 @@ function Public.control(config)
 
     local airstrike_data = {radius_level = 1, count_level = 1}
     local barrage_data = {radius_level = 1, count_level = 1}
+    local spy_message_cooldown = false
```

Comment:
This is desync risk. In lua Booleans are value types so this is a copy of the value stored in global. The trick is to store the value inside a table which are reference types e.g.
```suggestion
    local spy_message_cooldown = {false}
```
And then the places where this is set would be `spy_message_cooldown[1] = true/false`

Reply:
Cool. Thanks